src/System.Private.CoreLib/shared/System/Collections/ObjectModel/Collection.cs

```diff
@@ -125,12 +125,12 @@ public void InsertRange(int index, IEnumerable<T> collection)
 
             if (collection == null)
             {
-                ThrowHelper.ThrowArgumentNullException(ExceptionArgument.list);
+                ThrowHelper.ThrowArgumentNullException(ExceptionArgument.collection);
             }
 
             if ((uint)index > (uint)items.Count)
             {
-                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.index, ExceptionResource.ArgumentOutOfRange_ListInsert);
+                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.index, ExceptionResource.ArgumentOutOfRange_Index);
```

Comment:
I believe this is what `ThrowHelper.ThrowArgumentOutOfRange_IndexException();` is that's used elsewhere in this file.  Assuming it is, we should use that instead for both its terseness and consistency.

Reply:
I updated the exception handling for lines 133 and 113 as they are both are IndexOutOfRange Exceptions